assets/css/main.css

```diff
@@ -49,6 +49,57 @@ body {
   display: none;
 }
 
+/* mobile menu section */
+.mobile_menu {
+  width: 100%;
+  height: 100vh;
+  background-color: var(--dark-grey);
+  z-index: 1;
+  position: fixed;
+  top: -999px;
+  left: -999px;
+  transition: all linear;
+  transition-duration: 0.7s;
+}
+
+.mobile_menu-container {
+  height: 100%;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
- [ ] Well done, however, it would be better if your mobile menu design matches the given Figma's design. Because in your design the mobile menu options are centered while in Figma's design they are placed on the top left side. Take a look at the below screenshot of the original design and try to stick it 👇 
![Screenshot (244)](https://user-images.githubusercontent.com/67087664/161729522-f87c84c0-15a2-4a8a-8458-fc0de3c71e42.png)